infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -30,8 +30,13 @@
 public class BatchWriter<K, V> implements CacheWriter<K, V>, AutoCloseable {
 
     static final int BATCH_SIZE = 50;
+
+    private boolean doSync;
```

Comment:
To avoid someone modifying this on runtime, can this be a `final` variable?

Reply:
done.